parser.y

```diff
@@ -2429,6 +2434,10 @@ DatabaseOption:
 	{
 		$$ = &ast.DatabaseOption{Tp: ast.DatabaseOptionCollate, Value: $4.(string)}
 	}
+|	DefaultKwdOpt "ENCRYPTION" EqOpt EncryptionOption
```

Comment:
Let's just use `stringLit` here (and remove `EncryptionOption`). Like `TableOption` we normally don't create a separate rule for every option unless it is necessary.

```suggestion
|	DefaultKwdOpt "ENCRYPTION" EqOpt stringLit
```